src/Config/Config.js

```diff
@@ -25,6 +25,7 @@ export default {
     searchButton: 'searchButton',
     shareButton: 'shareButton',
     editPagesButton: 'editPagesButton',
+    showViewLayersToolbarButton: 'showViewLayersToolbarButton',
```

Comment:
showViewLayers

Reply:
Does that not break the naming convention of the other options ?

Reply:
good point, showViewLayersButton